engine-transactions/src/lib.rs

```diff
@@ -65,7 +66,7 @@ impl<'a> From<&'a EthTransactionKind> for Vec<u8> {
 /// A normalized Ethereum transaction which can be created from older
 /// transactions.
 pub struct NormalizedEthTransaction {
-    pub address: Option<Address>,
+    pub address: Address,
```

Comment:
As far as I remember, this `NormalizedEthTransaction` struct was inspired by a corresponding definition in geth. If the option is used in geth, does that mean there is a use case where we want to be able to define a transaction without a valid `from` address? I can't think of any myself, but I thought I would pose the question anyway.

Reply:
It wasn't an option. I believe an `Option` was lazily used here not to make an error type.